src/server/migrations/favorites_table.js

```diff
@@ -0,0 +1,15 @@
+exports.up = function (knex) {
```

Comment:
Please name your migration with the timestamp name. Please use these instructions As an example, this means you would have to run knex -- migrate:latest instead of knex migrate:latest and knex -- migrate make test_migration rather than knex migrate make test_migration. (https://github.com/HackYourFuture-CPH/fp-class18/blob/develop/knex-and-migrations.md)